packages/lesmis-server/src/app/createApp.ts

```diff
@@ -61,15 +57,12 @@ export function createApp() {
     /**
      * POST
      */
-
     .post<RegisterRequest>('register', handleWith(RegisterRoute))
     .post<ReportRequest>('report/:entityCode/:reportCode', handleWith(ReportRoute))
     .post<PDFExportRequest>('pdf', handleWith(PDFExportRoute))
 
+    .use('*', forwardRequest(CENTRAL_API_URL))
     .build();
 
-  // Forward any unhandled request to central-server
-  useForwardUnhandledRequests(app, CENTRAL_API_URL, '/admin', attachSession);
```

Comment:
It's safe to remove this custom handling for stripping `/admin` from the route, as LESMIS no longer uses the `lesmis-server` as  the backend for the LESMIS Admin Panel (it uses `admin-panel-server`)